src/components/measurements/measurementsD3.js

```diff
@@ -283,23 +285,36 @@ export const changeMeasurementsDisplay = (ref, display) => {
   });
 };
 
-export const setHoverTransition = (ref, handleHoverMeasurement) => {
+export const setHoverTransition = (ref, handleHover) => {
   const svg = select(ref);
+  // Handle hovers of raw measurements
   svg.selectAll(`.${classes.rawMeasurements}`)
     .on("mouseover", (d, i, elements) => {
       select(elements[i]).transition()
         .duration("100")
         .attr("r", layout.circleHoverRadius);
+
+      // Get mouse position for HoverPanel
+      const { clientX, clientY } = d3event;
       // sets hover data state to trigger the hover panel display
-      handleHoverMeasurement(d);
+      handleHover(d, "measurement", clientX, clientY);
     })
     .on("mouseout", (_, i, elements) => {
       select(elements[i]).transition()
         .duration("200")
         .attr("r", layout.circleRadius);
       // sets hover data state to null to hide the hover panel display
-      handleHoverMeasurement(null);
+      handleHover(null);
     });
+
+  // Handle hovers of mean and standard deviations
+  svg.selectAll(`.${classes.mean}, .${classes.standardDeviation}`)
```

Comment:
[no changes needed] For what it's worth, CSS selection can be a decent botteneck. A way around this is to store references to the SVG groups (as we do elsewhere in auspice) and then `svgGroup.selectAll('*')`. Without actually testing I couldn't say if it would noticeably change things (I'd guess not in this case, there's not that many elements per row and each row is its own SVG).